app/src/main/java/com/animal_villa/GameFramework.kt

```diff
@@ -6,12 +6,13 @@ import androidx.activity.ComponentActivity
 import com.example.main_animal_villa.R
 //import java.lang.reflect.Modifier
 
-class GameFramework : ComponentActivity() {
+public class GameFramework : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
     }
 }
+// Changes made to indentation
 
         /* {
             AnimalVillaTheme {
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.